database/mssql/scripts/versions/revert/v_21_ddl_revert.sql

```diff
@@ -0,0 +1,45 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+ALTER TABLE [permit].[ORBC_PERMIT_DATA] 
+ DROP COLUMN VIN
+
+
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [permit].[ORBC_PERMIT_DATA] 
+ ADD [APPLICANT]  AS (CONCAT (json_value([PERMIT_DATA],'$.contactDetails.firstName'), ' ', json_value([PERMIT_DATA],'$.contactDetails.lastName')))
+
```

Comment:
Should restore the MS_Description of APPLICANT column here

Reply:
The original v4 did not have a description for the col.